app/plugin_api.go

```diff
@@ -536,6 +536,16 @@ func (api *PluginAPI) GetGroupByName(name string) (*model.Group, *model.AppError
 	return api.app.GetGroupByName(name, model.GroupSearchOpts{})
 }
 
+func (api *PluginAPI) GetGroupMemberUsers(groupID string, page, perPage int) ([]*model.User, *model.AppError) {
+	users, _, err := api.app.GetGroupMemberUsersPage(groupID, page, perPage)
```

Comment:
Opted to only expose the paginated version vs. the "get all" given the potential size of groups (vs. the number of groups below).